src/Games/NexusMods.Games.RedEngine/Cyberpunk2077/Emitters/ArchiveXLMissingEmitter.cs

```diff
@@ -0,0 +1,26 @@
+using NexusMods.Abstractions.GameLocators;
+using NexusMods.Abstractions.Games.DTO;
+using NexusMods.Abstractions.NexusWebApi.Types;
+using NexusMods.Paths;
+
+namespace NexusMods.Games.RedEngine.Cyberpunk2077.Emitters;
+
+/// <summary>
+/// Diagnostic emitter for missing ArchiveXL when a mod installs .xl files in the game folder.
+/// </summary>
+public class ArchiveXLMissingEmitter : APathBasedDependencyEmitterWithNexusDownload
+{
+    protected override string DependencyName => "ArchiveXL";
+
+    protected internal override IEnumerable<GamePath> DependencyPaths => new[]
+    {
+        // ArchiveXL DLL
+        new GamePath(LocationId.Game, "red4ext/plugins/ArchiveXL/ArchiveXL.dll"),
+    };
+
+    protected internal override GamePath[] DependantPaths => [new GamePath(LocationId.Game, "")];
+    
+    protected internal override Extension[] DependantExtensions => [new(".xl")];
```

Comment:
```csharp
DependantExtensions { get; } = [new Extension(".xl")];
```

`=>` and `{ get; } =` work slightly different. See [this](https://sharplab.io/#v2:EYLgtghglgdgPgAQEwEYCwAoBBmABM3AYVwG9NcL88EUAGAbQF1cAVAC1gHMBnJUzgKYAXANwBfXAF5c9AEQAzAPaLZAGlyzgEAE6zGI8pRz46TVhxg8pAPhkLlajVt37DFN1XwAWXAFkAFACUpB5imGJAA=) for details.